androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/mediaRouter/MediaRouter.kt

```diff
@@ -0,0 +1,26 @@
+package com.profusion.androidenhancedvideoplayer.components.mediaRouter
+
+import android.content.Context
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.google.android.gms.cast.framework.CastContext
+
+@Composable
+fun MediaRouter(context: Context, modifier: Modifier = Modifier) {
+    val viewModel = MediaRouterViewModel(context)
+    val castContext = CastContext.getSharedInstance()
+
+    if (castContext != null) {
+        Box(modifier = modifier) {
+            AndroidView(
+                factory = { viewModel.mediaRouteButton() },
+                modifier = Modifier.size(36.dp)
```

Comment:
I think you should use `LocalDimensions`

Reply:
✅